codegen/src/main.rs

```diff
@@ -117,8 +118,19 @@ fn codegen(
 }
 
 fn write_fds(fds: &FileDescriptorSet, path: &Path) {
-    let mut writer = BufWriter::new(File::create(path).unwrap());
+    const GENERATED_COMMENT: &str = "// This file is @generated by codegen.";
+
     let mut buf = BytesMut::with_capacity(fds.encoded_len());
     fds.encode(&mut buf).unwrap();
-    writer.write_all(&buf).unwrap();
+
+    let fds_raw = buf.as_ref();
+    let tokens = quote! {
+        /// Byte encoded FILE_DESCRIPTOR_SET.
+        pub const FILE_DESCRIPTOR_SET: &[u8] = &[#(#fds_raw),*];
+    };
+
+    let content = format!("{GENERATED_COMMENT}\n{tokens}\n",);
+
+    let mut writer = BufWriter::new(File::create(path).unwrap());
+    writer.write_all(content.as_bytes()).unwrap();
```

Comment:
Maybe do some formatting, either with prettyplease or `rustfmt`?

Reply:
I've considered and tried formatting, but in this case, it does not seem helpful for up as they are mere slices of bytes resulting in a lot of newlines by each elements.

Reply:
Okay. IMO reviewing future changes to these files will probably still be easier if they're formatted more like "normal" Rust files.

Reply:
Sounds reasonable to me. Changed to format them.